cmd/ore/openstack/create.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openstack
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/coreos/mantle/sdk"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdCreate = &cobra.Command{
+		Use:   "create-image",
+		Short: "Create image on OpenStack",
+		Long: `Upload an image to OpenStack.
+
+After a sucessful run, the final line of output will be the id of the image.
+`,
+		RunE: runCreate,
+	}
+
+	path string
+	name string
+)
+
+func init() {
+	OpenStack.AddCommand(cmdCreate)
+	cmdCreate.Flags().StringVar(&path, "file",
+		sdk.BuildRoot()+"/images/amd64-usr/latest/coreos_production_openstack_image.img",
```

Comment:
Just a note, we should start thinking about how this type of stuff intersects with coreos-assembler which has a different build layout and naming convention.  But doesn't block this.